src/utils.ts

```diff
@@ -285,3 +285,9 @@ export function replaceKeys(obj: Record<string, any>): Record<string, any> {
     return obj;
   }
 }
+
+export function getEntries<Item>(obj: ArrayLike<Item> | Record<string, Item>, options: GlobalContext) {
+  const entries = Object.entries(obj);
+  if (options.alphabetize) entries.sort(([a], [b]) => a.localeCompare(b, "en"));
```

Comment:
👍 This is a smart place to alphabetize. Sorting too often could really slow things down. But for practical purposes I’d guess this sorts as little as possible if done on retrieval like this.